moabb/pipelines/utils.py

```diff
@@ -50,6 +50,31 @@ def create_pipeline_from_config(config):
     return pipeline
 
 
+def parse_Deeplearning_callbacks(config):
+    config_parsed = config
+
+    for j in np.arange(len(config["pipeline"])):
+        if "parameters" in config["pipeline"][j]:
+            if "callbacks" in config["pipeline"][j]["parameters"]:
+                callbacks = []
+                mod = __import__(config["pipeline"][j]["from"], fromlist=["funct_parser"])
+                funct_parser = mod.funct_parser
+
+                def parse_DL_param(funct):
+                    my_func = funct_parser[str(funct)]
+                    return my_func
```

Comment:
I understand that the function `parse_DL_parm` needs the variable `funct_parser`, which is being defined globally, but I think it's not a good practice not to pass all parameters as a parameter.

You can do something like this and move the function outside the loop.

```suggestion
                funct_parser = {}
                def parse_deep_param(funct, funct_parser=funct_parser):
                    my_func = funct_parser[str(funct)]
                    return my_func
```